flink-table-store-connector/src/test/java/org/apache/flink/table/store/connector/ReadWriteTableITCase.java

```diff
@@ -1224,6 +1226,144 @@ public void testQueryContainsDefaultFieldName() throws Exception {
                 .containsOnly(changelogRow("+I", 1, "abc"));
     }
 
+    @Test
+    public void testLike() throws Exception {
+        rootPath = TEMPORARY_FOLDER.newFolder().getPath();
+        tEnv = StreamTableEnvironment.create(buildBatchEnv(), EnvironmentSettings.inBatchMode());
+        List<Row> input =
+                Arrays.asList(
+                        changelogRow("+I", 1, "test_1"),
+                        changelogRow("+I", 2, "test_2"),
+                        changelogRow("+I", 1, "test_%"),
+                        changelogRow("+I", 2, "test%2"),
+                        changelogRow("+I", 3, "university"),
+                        changelogRow("+I", 4, "very"),
+                        changelogRow("+I", 5, "yield"));
+        String id = registerData(input);
+        UUID randomSourceId = UUID.randomUUID();
+        tEnv.executeSql(
+                String.format(
+                        "create table `helper_source_%s` (f0 int, f1 string) with ("
+                                + "'connector' = 'values', "
+                                + "'bounded' = 'true', "
+                                + "'data-id' = '%s')",
+                        randomSourceId, id));
+
+        UUID randomSinkId = UUID.randomUUID();
+        tEnv.executeSql(
+                String.format(
+                        "create table `managed_table_%s` with ("
+                                + "'%s' = '%s'"
+                                + ") like `helper_source_%s` "
+                                + "(excluding options)",
+                        randomSinkId, FileStoreOptions.PATH.key(), rootPath, randomSourceId));
+        tEnv.executeSql(
```

Comment:
To test the filter, the best way is to insert it multiple times so that multiple files are generated to achieve the effect we want to filter the files.